### PR TITLE
Added basic home page view functionality. Added nav bar and tests.

### PR DESCRIPTION
**Pull request by nagerz**

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<div class="home-roulette-now">
+  <h5>Ready to eat?</h5>
+  <%= button_to 'Roulette Now', restaurant_path, method: :get %>
+</div>
+<div class="home-refine-search">
+  <h5>Something in mind?</h5>
+  <%= button_to 'Select Options', refine_path, method: :get %>
+</div>
+<div class="home-survey">
+  <h5>Want input from a group?</h5>
```
Comment (juliamazey) [inline app/views/home/index.html.erb:20]: I actually liked the 'Picky friends' better, or something like that, more catchy??

File: app/views/partials/_nav.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<nav class="primary_nav">
```
Comment (juliamazey) [inline app/views/partials/_nav.html.erb:1]: should this be in the app layout? Since it's gonna show everywhere

File: config/routes.rb [hunk 3]
```diff
@@ -3,6 +3,13 @@
   get '/login', to: 'login#index'
   get 'auth/google', as: :google_connect
   get 'auth/google/callback',  to: 'sessions#create'
+
   get '/restaurant', to: 'restaurants#show', as: :restaurant
+  get '/refine', to: 'refine#show', as: :refine
```
Comment (juliamazey) [inline config/routes.rb]: I don't think we need to specify an alias if it's called the same as the path, Rails magic!

File: spec/features/nav_bar_spec.rb [hunk 4]
```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.feature 'Navigation bar' do
+  context 'as a visitor' do
+    context 'when I am redirected to the login page' do
+      it 'I do not see a navigation bar' do
```
Comment (juliamazey) [inline spec/features/nav_bar_spec.rb:6]: Oh, I see why the partial for the nav bar now...

**Review by juliamazey — APPROVED** [review]
I think we can merge this!

File: app/views/partials/_nav.html.erb (hunk 2 above)
Comment (nagerz) [reply app/views/partials/_nav.html.erb:1]: Not quite everywhere. Like not the log in page... Maybe better with an if statement, we can discuss

**Pull request merged**